internal/local/structure_v2.go

```diff
@@ -241,6 +253,24 @@ func parseFunctionsV2(rootDir string) (FunctionsStructure, error) {
 	return FunctionsStructure{configs, sources}, nil
 }
 
+func parseEndpointsV2(rootDir string) (EndpointStructure, error) {
```

Comment:
[question] where did we land on endpoints being for all config versions?  I still think its an additive change here, but I see in the backend `/import` supports it but `/export` does not

Reply:
I agree, it would be additive. When I last spoke with product it was not a requirement to add support for other config versions. Though I am open to discussing this change in the next check in since the disconnect between import/export support could be confusing

Reply:
yeah my main thing is, 6 months from now, it's going to be _more_ confusing than not if we don't just have it fully coded out.  it just ends up being another wrinkle to the overall differences b/w config versions that's going to need to be unwound once we tackle the refactor.

please file ticket regardless (and mark the `TODO` here) ... we can address how it to triage it in the check-in

Reply:
done! created https://jira.mongodb.org/browse/REALMC-10879, will add detail to the ticket